src/conv-core/convcore.C

```diff
@@ -1656,6 +1656,7 @@ void CsdSchedulerState_new(CsdSchedulerState_t *s)
 #if CMK_NODE_QUEUE_AVAILABLE
 	s->nodeQ=CsvAccess(CsdNodeQueue);
 	s->nodeLock=CsvAccess(CsdNodeQueueLock);
+	s->nodeGrpFreq =2; // check node queue once in 4 (2^2) iterations. 
```

Comment:
Should this be configurable by the user?

Reply:
Looks like i forgot about this PR once I traveled to India. To answer your question: ideally, yes. And we will make it user configurable when we do registration-based/table-driven scheduler for managing queues. But for now, 4 seems like a reasonable ad-hoc number. Adding another build time option for a feature that will be replaced in near future seems like an overkill.